.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -82,36 +82,30 @@ runs:
             cd -  # Return to the previous directory
         done
 
-        cd ./chart
+        cd .github/actions/deploy-klt-on-cluster
 
-        touch tmp-values.yaml
         if [ "${{ inputs.allowed-namespaces }}" == "allowed_ns_on" ]; then
-          echo "lifecycleOperator:" >> tmp-values.yaml
-          echo "  allowedNamespaces: [allowed-ns-test]" >> tmp-values.yaml
+          echo "lifecycleOperator:" >> values.yaml
+          echo "  allowedNamespaces: [allowed-ns-test]" >> values.yaml
         fi
 
         if [ "${{ inputs.scheduling-gates }}" == "gates_on" ]; then
           if [ "${{ inputs.allowed-namespaces }}" == "allowed_ns_off" ]; then
-            echo "lifecycleOperator:" >> tmp-values.yaml
+            echo "lifecycleOperator:" >> values.yaml
           fi
-          echo "  schedulingGatesEnabled: true" >> tmp-values.yaml
+          echo "  schedulingGatesEnabled: true" >> values.yaml
         fi
+        echo "installing with values.yaml file:"
+        cat values.yaml
+
+        cd ../../../chart
```

Comment:
you can also just use the path in the `helm install` cmd. then the folder hopping could be reduced